scrapers/AmateureExtreme.yml

```
@@ -0,0 +1,32 @@
+name: "AmateureXtreme"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - amateure-xtreme.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class='info-section--title']
+      URL: //link[@rel="canonical"]/@href
+      Date:
+        selector: //div[@class='posted-info meta-description small']/div[@class='float-left']/span[@class='text-nowrap']/span[@class='fa5-text']
+        postProcess:
+          - replace:
+              - regex: $
+                with: $1-01-01

```

```suggestion
                with: -01-01
```
No match group is used so `$1` doesnt have a value anyway

---

Thank you.